client/components/jetpack/fresh-start-2021-sale-banner/index.tsx

```diff
@@ -31,15 +31,15 @@ type TimeWindowProps = {
 	end: Moment;
 };
 
-const TimeGatedNewYear2021SaleBanner: React.FC< Props > = ( { urlQueryArgs } ) => {
-	const paramValue = urlQueryArgs?.[ 'newpack' ];
-
+const TimeGatedFreshStart2021SaleBanner: React.FC< Props > = ( { urlQueryArgs } ) => {
+	const paramValue = urlQueryArgs?.[ 'freshstart' ];
 	const moment = useLocalizedMoment();
-	if ( paramValue === undefined ) {
-		// Start at midnight UTC on Jan 1,
-		// and continue until 23:59:59 UTC on Jan 18
-		const start = moment.utc( '2021-01-01' );
-		const end = moment.utc( '2021-01-19' );
+
+	if ( ! paramValue ) {
+		// Start at midnight UTC on Feb 1,
+		// and continue until 23:59:59 UTC on Feb 14
+		const start = moment.utc( '2021-02-01' );
+		const end = moment.utc( '2021-02-14' );
```

Comment:
If I'm reading this correctly, the promo banner will stop being visible as soon as the system clock reads 2021-01-14, meaning the moment directly after 23:59:59 on February 13. If I understood correctly, we want this to be visible thru the last moment of February 14, so `end` should be like so:

```suggestion
		const end = moment.utc( '2021-02-15' );
```

Reply:
Good catch, thanks!